ScoutSuite/providers/aws/services/awslambda.py

```diff
@@ -1,34 +1,39 @@
 # -*- coding: utf-8 -*-
-"""
-Lambda-related classes and functions
-"""
-
-from ScoutSuite.providers.aws.configs.regions import RegionalServiceConfig, RegionConfig, api_clients
-
-
-
-########################################
-# LambdaRegionConfig
-########################################
-
-class LambdaRegionConfig(RegionConfig):
-
-    def parse_function(self, global_params, region, function):
+from ScoutSuite.providers.base.configs.resource_config import ResourceConfig
+from ScoutSuite.providers.aws.configs.regions_config import RegionsConfig
+from ScoutSuite.providers.aws.aws_facade import AwsFacade
+from opinel.utils.aws import build_region_list
+
+class LambdaServiceConfig(RegionsConfig):
```

Comment:
Since this classe stores lambdas accross all regions, I would name it `Lambdas`.